src/DDTrace/Integrations/Lumen/V5/LumenIntegrationLoader.php

```diff
@@ -0,0 +1,129 @@
+                $resourceName = $this->currentRoute[1]['as'];
+            }
+            $span->setTag(Tag::RESOURCE_NAME, $span->getTag(Tag::HTTP_METHOD) . ' ' . $resourceName);
+            return $response;
+        });
+
+        dd_trace('Symfony\Component\HttpFoundation\Response', 'setStatusCode', function ($code) use ($span) {
+            $span->setTag(Tag::HTTP_STATUS_CODE, $code);
+            return dd_trace_forward_call();
+        });
+
+        // Trace views
+        dd_trace('Illuminate\View\Engines\CompilerEngine', 'get', function () {
+            $scope = GlobalTracer::get()->startIntegrationScopeAndSpan(
+                LumenIntegration::getInstance(),
+                'lumen.view'
+            );
+            $scope->getSpan()->setTag(Tag::SPAN_TYPE, Type::WEB_SERVLET);
+            return include __DIR__ . '/../../../try_catch_finally.php';
+        });
+
+        // Trace middleware
+        $traceMiddleware = function ($middlewares) {
+            foreach ($middlewares as $middleware) {
+                // Ignore closures
+                if ($middleware instanceof \Closure) {
+                    continue;
+                }
+                if (is_object($middleware)) {
+                    $middleware = get_class($middleware);
+                }
+                if (!is_string($middleware)) {
+                    continue;
+                }
+                // Middleware can be passed parameters during registration, in the form
+                // 'middleware_name_or_class:param1,param2', so we need to extract the real name/class from the
+                // pipeline
+                // See: https://laravel.com/docs/5.7/middleware#middleware-parameters
+                $middleware = explode(':', $middleware)[0];
+                dd_trace($middleware, 'handle', function () {
+                    $tracer = GlobalTracer::get();
+                    if ($tracer->limited()) {
+                        return dd_trace_forward_call();
+                    }
+                    $scope = $tracer->startIntegrationScopeAndSpan(
+                        LumenIntegration::getInstance(),
+                        'lumen.pipeline.pipe'
+                    );
+                    $span = $scope->getSpan();
+                    $span->setTag(Tag::RESOURCE_NAME, get_class($this) . '::handle');
+                    $span->setTag(Tag::SPAN_TYPE, Type::WEB_SERVLET);
+                    return include __DIR__ . '/../../../try_catch_finally.php';
+                });
+            }
+
+            return dd_trace_forward_call();
+        };
+        dd_trace('Laravel\Lumen\Application', 'middleware', $traceMiddleware);
+        dd_trace('Laravel\Lumen\Application', 'routeMiddleware', $traceMiddleware);
+
+        return Integration::LOADED;
+    }
+
+    /**
+     * @return bool
+     */
+    private function shouldLoad()
+    {
+        if ('cli' === PHP_SAPI && 'dd_testing' !== getenv('APP_ENV')) {
```

Comment:
I think that this is outdated correct? :)

Reply:
It will be when #422 is merged - I can delete this from a rebase of that branch if that's OK with you. :)